src/core/sys/posix/pthread.d

```diff
@@ -209,6 +209,46 @@ else version( FreeBSD )
     enum PTHREAD_COND_INITIALIZER               = null;
     enum PTHREAD_RWLOCK_INITIALIZER             = null;
 }
+else version( DragonFlyBSD )
+{
+    enum
+    {
+        PTHREAD_DETACHED            = 0x1,
+        //PTHREAD_SCOPE_SYSTEM        = 0x2,    // defined below
+        PTHREAD_INHERIT_SCHED       = 0x4,
+        PTHREAD_NOFLOAT             = 0x8,
+
+        PTHREAD_CREATE_DETACHED     = PTHREAD_DETACHED,
+        PTHREAD_CREATE_JOINABLE     = 0,
+        //PTHREAD_SCOPE_PROCESS       = 0,      // defined below
+        PTHREAD_EXPLICIT_SCHED      = 0,
+    }
+
+    enum
+    {
+        PTHREAD_PROCESS_PRIVATE     = 0,
+        PTHREAD_PROCESS_SHARED      = 1,
+    }
+
+    enum
+    {
+        PTHREAD_CANCEL_ENABLE       = 0,
+        PTHREAD_CANCEL_DISABLE      = 1,
+        PTHREAD_CANCEL_DEFERRED     = 0,
+        PTHREAD_CANCEL_ASYNCHRONOUS = 2,
+    }
+
+    enum PTHREAD_CANCELED = cast(void*) -1;
+
+    enum PTHREAD_NEEDS_INIT = 0;
+    enum PTHREAD_DONE_INIT  = 1;
+
+    enum PTHREAD_MUTEX_INITIALIZER              = null;
+    //enum PTHREAD_ONCE_INIT                      = { PTHREAD_NEEDS_INIT, NULL };
+    enum PTHREAD_ONCE_INIT                      = pthread_once_t.init;;    
```

Comment:
Delete one semi-colon, missed this upstream too.

Reply:
Done